docs/src/router/drawer.tsx

```diff
@@ -136,6 +148,41 @@ export const NavigationDrawer: React.FC = () => {
                         items={convertNavItems(navGroup.pages || [], navGroup.path || '', 0, handleNavigate, dispatch)}
                     />
                 ))}
+                <DrawerNavGroup
+                    title="COMMUNITY"
+                    titleColor={Colors.blue[500]}
```

Comment:
is it possible to put `"primary.main"` here instead of a direct color reference?

Reply:
We are not using sx prop in DrawerNavGroup instead we are using style, so passing a string will not parse. 
<img width="794" alt="Screenshot 2023-03-23 at 1 47 29 PM" src="https://user-images.githubusercontent.com/120575281/227147594-56f9385e-ee38-493a-9ff6-2680bb486f94.png">